packages/iov-cli/README.md

```diff
@@ -200,80 +200,22 @@ When using a Testnet, you can use the IovFaucet to receive tokens:
 'helmet album grow detail apology thank wire chef fame core private cargo'
 > const profile = new UserProfile();
 > const wallet = profile.addWallet(Ed25519HdWallet.fromMnemonic(mnemonic));
-> const me = await profile.createIdentity(wallet.id, HdPaths.simpleAddress(0));
 
 > const signer = new MultiChainSigner(profile);
-> const { connection } = await signer.addChain(bnsConnector("https://bns.yaknet.iov.one"));
-> const meAddress = signer.keyToAddress(connection.chainId(), me.pubkey);
+> const { connection } = await signer.addChain(bnsConnector("https://bns.hugnet.iov.one"));
+> const chainId = connection.chainId();
 
-> const faucet = new IovFaucet("https://iov-faucet.yaknet.iov.one");
+> const me = await profile.createIdentity(wallet.id, chainId, HdPaths.simpleAddress(0));
+> const meAddress = signer.identityToAddress(me);
```

Comment:
`myAddress`?

Reply:
Yeah, renamed to alice and aliceAddress to avoid the naming strangeness